src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Options/PackageSourcesPage.cs

```diff
@@ -74,6 +75,8 @@ public override async Task<ExternalSettingOperationResult> SetValueAsync<T>(stri
                 throw new InvalidOperationException();
             }
 
+            bool hasAnyHiddenPropertyChanged = false;
```

Comment:
Can you help me understand what a hidden property is?

Reply:
One that is not shown in the UI. We pass it to USX and it will pass it back to NuGet. But it's never rendered.